syft/frameworks/torch/hook/hook.py

```diff
@@ -540,7 +540,7 @@ def overloaded_attr(self, *args, **kwargs):
             """
 
             # Replace all syft tensor with their child attribute
-            new_self, new_args, new_kwargs = syft.frameworks.torch.hook_args.hook_method_args(
+            new_self, new_args, new_kwargs = syft.frameworks.torch.hook_args.hook_object_method_with_args(
```

Comment:
Maybe `unwrap_args_in_method` would be more accurate (or `hook_args_in_method` but maybe less clear)

Reply:
`hook_method_args` function was hooking the method of object with args so I named it as `hook_object_method_with_args` in layman terms. `unwrap_args_in_method` would still be a bit confusing about the its functionality. If you want, we could name it `unwrap_args_with_method`